CONTRIBUTING.md

```diff
@@ -360,3 +361,56 @@ The following command is used for running the tasks: `grunt [task-name]`. Here `
 4. **serve**: Compiles the dist files as in the **compile** task and starts a local server on `localhost:3000` to host the demo site in `/examples/` directory. Also runs the **watch** task.
 5. **production**: Compiles and minifies dist files in `/dist/image-sequencer.js` and `/dist/image-sequencer-ui.js` without the `.min.js` extension to include minified files in the demo site. This script should only be used in production mode while deploying.
 6. **default**: Runs the **watch** task as default.
+
+## UI Helper Methods
+
+### scopeQuery
+
+###### Path: `/examples/lib/scopeQuery.js`
+
+The method returns a scoped `jQuery` object which only searches for elements inside a given scope (a DOM element).
+
+To use the method, 
+* import the `scopeSelector` and `scopeSelectorAll` methods from `lib/scopeQuery.js`
+* call the methods with scope 
+	
+```js
+var scopeQuery = require('./scopeQuery');
+
+var $step = scopeQuery.scopeSelector(scope),
+ 	  $stepAll = scopeQuery.scopeSelectorAll(scope);	
+```
+This will return an object with a constructor which returns a `jQuery` object (from inside the scope) but with new `elem` and `elemAll` methods.
+
+#### Methods of the Returned Object
+* `elem()`: selects an element inside the scope; 
```

Comment:
Try to capitalize wherever possible.(only sentences, do not capitalize anything in any code.

Reply:
You mean to capitalize the words in the title? Articles and prepositions shouldn't be capitalized in titles, so grammatically the title is correct. Is it some sort of internal rule to capitalize every single word?

Reply:
I mean if there is something which isn't capitalized(only the first letter of a title) and if there isn't a rule which doesn't let us capitalize it, capitalize it. Only the first letter of a title(if not capitalized)

Reply:
Ok, I misunderstood you at first.